File: Sistema/src/app/models/PessoaConsultorioRetorno.Model.ts
export class PessoaConsultorioRetornoModel {
  pesCodi: number;
  pesCPF: string;
  pesNome: string;
  tipDesc: string;
  pesStat: boolean;

  constructor() {
    this.pesCodi = 0;
    this.pesCPF = "";
    this.pesNome = "";
    this.tipDesc = "";
    this.pesStat = false;
  }
}

File: API/Models/PessoaConsultorioRetornoModel.cs
namespace API_BetterLife.Models
{
    public class PessoaConsultorioRetornoModel
    {
        public long pesCodi { get; set; }
        public string? pesCPF { get; set; }
        public string? pesNome { get; set; }
        public string? tipDesc { get; set; }
        public bool pesStat { get; set; }
    }
}
